data/templates/test_report.html

```diff
@@ -135,6 +139,24 @@
                             <span class="label label-success pull-right">XFAILED</span>
                         {% endif %}
                     {% endif %}
+                    {% if test.composite %}
+                    <br>
+                    Streak
+                        {% if test.outcomes['overall'] == "passed" %}
```

Comment:
This is cool for now, I really want to make these more usable, and probably have a dict so we don't need to keep having these case lists.